### ISSUE TITLE
Invalid amount parameter with values over $1,500

### ISSUE THREAD

**Issue by rodjsta** [cartalyst/stripe-laravel#78]
- PHP Version: 7.3
- Stripe Laravel Version: 12.0

Since this weekend past, Stripe is returning an `invalid_request_error` stating that the `amount` parameter is invalid and that amounts must be less than $1,500 AUD. All amounts less than $1,500 are being processed successfully.

I have included a copy of the response log and also a copy of the request for a failed request.

I cannot find anywhere in Stripe where a limit might be set.

Any help would be appreciated.

<img width="722" alt="image" src="https://user-images.githubusercontent.com/32949856/174524422-a0fbd9d7-0624-446b-9fdd-9dbff8ddbbd7.png">

<img width="524" alt="image" src="https://user-images.githubusercontent.com/32949856/174524677-e837d53d-7079-4188-8ca8-b2d189b20094.png">

**Comment by brunogaspar** [comment]
Hello @rodjsta 

I'm unsure what might be causing it, can you check on your Stripe Dashboard if there's anything obvious? Otherwise try reaching out to Stripe support and see if they have any kind of insight about what have changed.

Sorry for not being more helpful right now.

**Comment by GrahamCampbell** [comment]
This is probably happening because your Stripe account is not fully verified. You likely need to provide all your tax and billing information.

**Comment by brunogaspar** [comment]
Yup, i think what @GrahamCampbell mentioned or something around that area to be the culprit.

Going to close for now, but, please do ping if you can't figure it out.

**Comment by rodjsta** [comment]
Hi @brunogaspar and @GrahamCampbell, thank you for replying. I did read your messages on the day, but I'm just coming back to close the loop here in case anybody else stumbles across this.

Without notice, Stripe put a hard limit on the amounts that were able to be processed on the account due to the transaction values (BECS and Credit Cards ~$500k per month). After discussing with Stripe, the limits have been increased to a more business appropriate amount.

Interestingly, my local (testing keys and different account, but the same app) still has the hard $1,500 limit. I haven't pursued, but I wonder if my local testing app was caught up in the association.

Anyway, I appreciated your speedy replies when I posted.
